CHANGELOG.md

```diff
@@ -10,7 +10,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ----------------------
 ### Added
 - **New icons:** Art Text 4 (`.art4` directories), Bikeshed (`.bs`), dotenv (`.env{,.*}`), Figma (`.figma`), Imgbot (`.imgbotconfig`), Mint (`.mint`), PhotoRec (`.photorec.cfg`), ShellCheck (`.shellcheckrc`), Sponge-powered Mixin (`.mixin{,s}.json`), Windi CSS (`windi.config.{ts,js}`)
-- **Support:** Appcelerator Titanium (`.tss`), BC (`.bc`, `bc.library`), Checksum lists (`{.,}{check,ck,crc{,32},md5,rmd160,sha{1..3,224,384,512}}sum{,s}`), DC (`.dc`, `.dcrc`), Database (`.tgn`), EditorConfig (`.ecrc`), Generic (`.ed`), Git (`.lfsconfig`), Grammar (`.grammar`), Mail (`.eml`, `.mbox`, `.mbx`), Maps (`.mapping`, `.tiny`, `.tinyv2`, `.enigma`, `.match`, `.unpick`), PEG.js (`.peggy`), Roff (`*.cstr`, `cstr.54`, `indep_tr`), TextMate (`.tm_properties`, `.tmProperties`), Units (`units.dat`, `unittab`), XML (`.xmp`), Yacc (`.ypp`)
+- **Support:** Appcelerator Titanium (`.tss`), BC (`.bc`, `bc.library`), Checksum lists (`{.,}{check,ck,crc{,32},md5,rmd160,sha{1..3,224,384,512}}sum{,s}`), DC (`.dc`, `.dcrc`), Database (`.tgn`), EditorConfig (`.ecrc`), Generic (`.ed`), Git (`.lfsconfig`), Grammar (`.grammar`), Heex (`.heex`, `.html.heex`), Mail (`.eml`, `.mbox`, `.mbx`), Maps (`.mapping`, `.tiny`, `.tinyv2`, `.enigma`, `.match`, `.unpick`), PEG.js (`.peggy`), Roff (`*.cstr`, `cstr.54`, `indep_tr`), TextMate (`.tm_properties`, `.tmProperties`), Units (`units.dat`, `unittab`), XML (`.xmp`), Yacc (`.ypp`)
```

Comment:
Since we don't have an icon for Heex, it makes more sense to describe this change as Elixir support:

~~~diff
-Heex (`.heex`, `.html.heex`)
+Elixir (`.heex`, `.html.heex`)
~~~